validator/coordinator/src/test/java/tech/pegasys/teku/validator/coordinator/BlockFactoryTest.java

```diff
@@ -73,6 +90,9 @@
   void setUp() {
     when(depositProvider.getDeposits(any(), any())).thenReturn(deposits);
     when(attestationsPool.getAttestationsForBlock(any())).thenReturn(attestations);
+    when(attesterSlashingPool.getItemsForBlock(any())).thenReturn(attesterSlashings);
+    when(proposerSlashingPool.getItemsForBlock(any())).thenReturn(proposerSlashings);
+    when(voluntaryExitPool.getItemsForBlock(any())).thenReturn(voluntaryExits);
```

Comment:
We should probably check that these actually get included.

Reply:
Good catch. Done.